### DISCUSSION TITLE
Exclusive Launch - Thank-You Card Series

### DISCUSSION THREAD

**Discussion by MaryTitan** [Titannet-dao/Community#7] [RetroPGF]
🌟Dear Titan Network Community Members,

💎We are so excited to launch our exclusive Thank-You Card series which features both our limited edition Dragon Card and Web3 Music Festival Card. This initiative is our way of expressing our gratitude for your continued support and contributions, which have been crucial to the growth of the Titan Network ecosystem.

🐉Dragon Thank-You Card Distribution:
- Cards numbered from 1 to 1000 are earmarked for members who have already devoted contributions. These will be distributed on a first-come, first-served basis by our global team of volunteers.

- Cards numbered from 1001-2000 are reserved for those who will be attending our upcoming live events. These cards include contribution credits to encourage further engagement with our [RetroPGF program.](https://github.com/Titannet-dao/Community/discussions/3)

🎁Reward Redemption:
- You can redeem your rewards using the QR code located on the back side of the card, which will be activated following the launch of our mainnet.

💓We can't wait to witness more active participation and contributions from our members. Together, let’s work together to amplify the growth and development of the Titan Network Community! 

Titan Network

![image](https://github.com/Titannet-dao/Community/assets/164492336/342f48b4-7e65-4c32-915b-13ef58423186[) 
![image](https://github.com/Titannet-dao/Community/assets/164492336/9afbb235-7767-4182-a23c-fb8020b54e32)

**Comment by yynbttkx** [comment]
参与 e_bea29277-2c06-496c-a0ef-7e5903d4ff42，e_84b2daeb-205c-48ac-ad75-2891340a5f2f，e_1531080a-89a6-4233-bc52-11e99bcf2dc4，e_2fd136a8-6502-49a1-b60e-39118add5763

**Reply by MaryTitan** [reply]
You are welcome to participate in our live events to receive the card. We will ho-cost the first web3 music festival in Bangkok in May. Please stay tuned.

**Comment by eddielueng** [comment]
也没有写明参加方式？第一个是按照加入节点的时间排序？那第二个呢？

**Reply by MaryTitan** [reply]
You are welcome to participate in our live events to receive the card. We will ho-cost the first web3 music festival in Bangkok in May. Please stay tuned.

**Comment by nqtuan1183** [comment]
Great!!!!

**Comment by nqtuan1183** [comment]
> 🌟Dear Titan Network Community Members,
> 
> 💎We are so excited to launch our exclusive Thank-You Card series which features both our limited edition Dragon Card and Web3 Music Festival Card. This initiative is our way of expressing our gratitude for your continued support and contributions, which have been crucial to the growth of the Titan Network ecosystem.
> 
> 🐉Dragon Thank-You Card Distribution:
> 
> * Cards numbered from 1 to 1000 are earmarked for members who have already devoted contributions. These will be distributed on a first-come, first-served basis by our global team of volunteers.
> * Cards numbered from 1001-2000 are reserved for those who will be attending our upcoming live events. These cards include contribution credits to encourage further engagement with our [RetroPGF program.](https://github.com/Titannet-dao/Community/discussions/3)
> 
> 🎁Reward Redemption:
> 
> * You can redeem your rewards using the QR code located on the back side of the card, which will be activated following the launch of our mainnet.
> 
> 💓We can't wait to witness more active participation and contributions from our members. Together, let’s work together to amplify the growth and development of the Titan Network Community!
> 
> Titan Network
> 
> ![image](https://private-user-images.githubusercontent.com/164492336/321587934-342f48b4-7e65-4c32-915b-13ef58423186.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjk3NTAyNjMsIm5iZiI6MTcyOTc0OTk2MywicGF0aCI6Ii8xNjQ0OTIzMzYvMzIxNTg3OTM0LTM0MmY0OGI0LTdlNjUtNGMzMi05MTViLTEzZWY1ODQyMzE4Ni5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDI0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAyNFQwNjA2MDNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05ZmFlMjBiNThlYTYzYmIxNDQ4ZGE3YjU4NmZlNDkwMzRmOTYzMTI1M2NmNTdkNDdmNzE5NTM1OWFhYWE0MDk1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.UhJQ-eS249wysL2vMnYZLuBnMbHiBU7wM5-N8nxkb2Y) ![image](https://private-user-images.githubusercontent.com/164492336/328887520-9afbb235-7767-4182-a23c-fb8020b54e32.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjk3NTAyNjMsIm5iZiI6MTcyOTc0OTk2MywicGF0aCI6Ii8xNjQ0OTIzMzYvMzI4ODg3NTIwLTlhZmJiMjM1LTc3NjctNDE4Mi1hMjNjLWZiODAyMGI1NGUzMi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDI0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAyNFQwNjA2MDNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zYmRiNmNlNjE3YTE2MmJmYzkxOWJiMjk2ZmZkYTI1MDE4OGJjMTIyMjZkMTc1YmQ1YzIyMDI2MTVjZTFjZDQyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.zYxHJ_Hsnx9r7EB3x2up-orWzScd-eFeGruDBvYzNUQ)

**Reply by yynbttkx** [reply]
何为最强

**Comment by endebeny** [comment]
Please send kartu exslusiv user id Email: endebeny1998@gmail.com

**Comment by Portal0g** [comment]
Great
